var/spack/repos/builtin/packages/cairo/package.py

```diff
@@ -37,7 +37,7 @@ class Cairo(Package):
     depends_on("glib")
     depends_on("pixman")
     depends_on("freetype")
-    depends_on("fontconfig@2.10.91:")  # Require newer version of fontconfig.
+    #depends_on("fontconfig@2.10.91:")  # Require newer version of fontconfig.
```

Comment:
Not sure; it did build as-is on my OS X machine. Now to go back through and see what's up with the comments I added long ago…